client/api/InnerClient.cs

```diff
@@ -415,6 +481,12 @@ public JObject JsonVariation(string key, Target target, JObject defaultValue)
                 }
             }
         }
+        
+        // Method to encapsulate setting the sdkInitialized variable
+        private void SetSdkInitialized(bool value)
```

Comment:
could be a private setter on the property

Reply:
Cool - done.